hub/src/test/java/ch/prevo/open/hub/match/MatcherTest.java

```diff
@@ -35,12 +62,15 @@ public void findMatchesEmptyInput() throws Exception {
 
     @Test(expected = RuntimeException.class)
     public void findMatchesWithDuplicates() throws Exception {
-        Set<InsurantInformation> exits = new HashSet<>();
-        exits.add(new InsurantInformation("oasi1", "uid1"));
-        Set<InsurantInformation> entries = new HashSet<>();
-        entries.add(new InsurantInformation("oasi1", "uid2"));
-        entries.add(new InsurantInformation("oasi1", "uid3"));
+        Set<InsurantInformation> exits = createSet(new InsurantInformation(OASI_1, UID_1));
+        Set<InsurantInformation> entries = createSet(new InsurantInformation(OASI_1, UID_2), new InsurantInformation(OASI_1, UID_3));
 
         new MatcherService().findMatches(exits, entries);
     }
+
+    private Set<InsurantInformation> createSet(InsurantInformation... insurantInformations) {
```

Comment:
If no one has a strong opinion against it, we could use guava. Then it would be as simple as `Sets.newHashSet(E... elements)`

Reply:
sure, let's go for it